startup/GafferAppleseedUI/appleseedExamples.py

```diff
@@ -51,3 +52,28 @@
 	]
 )
 
+GafferUI.Examples.registerExample(
+	"Rendering/Multi-shot Render Spreadsheet",
+	"$GAFFER_ROOT/resources/examples/multiShotRenderSpreadsheet.gfr",
```

Comment:
Can we move the files into the matching folders?

Reply:
202ed056e12259507a6864b11bcfae6ad716bd16